mobility_games/games/wall_audio_player.py

```diff
@@ -19,54 +19,67 @@ def __init__(self):
         self.dingsound = path.join(top, 'auditory/sound_files/ding.wav')
         srv = Server(WallAudioConfig, self.config_callback)
         #self.gmode = rospy.get_param('~pc_audio_gmode', 'pvr')
-        self.pitch = rospy.get_param('~pc_pitch', True)
-        self.vol = rospy.get_param('~pc_vol', True)
-        self.speed = rospy.get_param('~pc_rate', True)
-        self.track = rospy.get_param('~pc_track', False)
-        self.ding = rospy.get_param('~pc_ding', False)
-        self.randrange = rospy.get_param('~pc_randrange', 1);
-        self.quantizetype = rospy.get_param('~pc_quanttype', 'dom7')
-        self.key = rospy.get_param('~pc_key', 'A')
+        self.pitch = rospy.get_param('~pitch', True)
+        self.pitchdefault = rospy.get_param('~pitchDefault', 'A4')
+        self.vol = rospy.get_param('~vol', True)
+        self.voldefault = rospy.get_param('~volDefault', .8)
+        self.speed = rospy.get_param('~rate', True)
+        self.speeddefault = rospy.get_param('~rateDefault', 20);
+        self.track = rospy.get_param('~track', False)
+        self.ding = rospy.get_param('~ding', False)
+        self.randrange = rospy.get_param('~randRange', 0);
+        self.quantizetype = rospy.get_param('~quantType', 'M')
+        self.key = rospy.get_param('~key', 'A')
+        self.on = rospy.get_param('~on', False)
+        self.altsounddist = rospy.get_param('~successZoneSize', .5)
+        self.maxfreqdist = 5.0;
         self.last_sound_time = rospy.Time.now() #Initialize the last time a sound was made
         self.dist = None
 
         self.player = ac.player() #play the sound.
-        self.altsounddist = .5;
-        self.rate = 1.0/20.0;
+        self.rate = 1.0/self.speeddefault;
         self.r = rospy.Rate(1.0/self.rate);
+        self.dingHappened = False;
 
     def config_callback(self, config, level):
-        self.pitch = config['pc_pitch']
-        self.vol = config['pc_vol']
-        self.speed = config['pc_rate']
-        self.track = config['pc_track']
-        self.ding = config['pc_ding']
-        self.quantizetype = config['pc_quanttype']
-        self.key = config['pc_key']
-        self.randrange = config['pc_randrange']
+        self.pitch = config['pitch']
+        self.vol = config['vol']
+        self.speed = config['rate']
+        self.track = config['track']
+        self.ding = config['ding']
+        self.quantizetype = config['quantType']
+        self.key = config['key']
+        self.randrange = config['randRange']
+        self.on = config['on']
+        self.pitchdefault = config['pitchDefault']
+        self.voldefault = config['volDefault']
+        self.speeddefault = config['rateDefault']
         return config
 
     def get_dist(self, msg):
         self.dist = msg.data
+        print(self.dist)
         if self.speed:
             filtereddist =  max(min(3*self.dist, 20), 2)
             self.rate = 1.0/filtereddist;
             self.r = rospy.Rate(1.0/self.rate)
```

Comment:
`         if self.speed:`
`             filtereddist =  max(min(3*self.dist, 20), 2)`
`             self.rate = 1.0/filtereddist;`
`             self.r = rospy.Rate(1.0/self.rate)`

A litte confusing what it's doing here. Might what to comment what the min/max is doing (limiting range from 2 to 20). Commenting purpose of function might help too.